packages/ast/src/nodes.js

```diff
@@ -3,11 +3,11 @@
 // THIS FILE IS AUTOGENERATED
 // see scripts/generateNodeUtils.js
 
-function assert(cond: boolean) {
-  if (!cond) {
-    throw new Error("assertion error");
-  }
-}
+import { assert } from "mamacro";
+
+// FIXME(sven): compat with assert
+// eslint-disable-next-line no-unused-vars
+class RuntimeError extends Error {}
```

Comment:
The `assert` macro uses RuntimeError, which forces me to define it here. If i switch to `Error` into the macro many tests from the kernel/exec will fail, so I don't know what to do yet.

Maybe having two macros.